src/transforms/dropExtrema.jl

```diff
@@ -0,0 +1,27 @@
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+        DropExtrema(low, high)
```

Comment:
Try to follow the docstring conventions `Scale` transform as it also has a low and a high.

Reply:
I will update it sir similar to what is done in other transform methods , thanks for pointing out.